### PR TITLE
[#991] - Agregar tag de "bosquejo" para nuevos issues

### LINKED ISSUE #991: Agregar automáticamente el tag "bosquejo" a los issues nuevos
### Descripción

Dado que por lo general los issues se crean a partir de una idea inacabada del trabajo a realizar para, posteriormente, proceder a detallar con mayor precisión su contenido, alcance y criterios de aceptación, se hace necesario modificar todos los issue templates para que, automáticamente, sean creados con el tag `✍️ bosquejo`.

En caso de que la primera definición del issue sea detallada, el tag puede ser removido al momento de la creación.

### Recursos

Revisar definiciones de issue templates en los archivos `*.yml` dentro del directorio `.github/ISSUE_TEMPLATE`

### PR DESCRIPTION
**Pull request by Demon-Sheriff**
Fixes #991
This PR modifies the existing issue templates `(/.github/ISSUE_TEMPLATE/bug_report.yml)` to automatically assign the "bosquejo" (draft) label to all newly created issues.

### CODE REVIEW

**Review by rolivencia — CHANGES_REQUESTED** [review]
It looks good, but we have to remove the changes made in `navigable-story-teaser.component.ts`, which don't belong here.

Once you get rid of that, we can approve the PR and merge it.

File: src/app/components/navigable-story-teaser/navigable-story-teaser.component.ts [hunk 1]
```diff
@@ -46,8 +47,8 @@ import { RouterLink } from '@angular/router';
 	`,
 })
 export class NavigableStoryTeaserComponent {
-	story = input.required<StoryTeaser>();
-	selected = input<boolean>();
-	authorSlug = input.required<string>();
+	@Input() story!: StoryTeaser;
```
Comment (rolivencia) [inline src/app/components/navigable-story-teaser/navigable-story-teaser.component.ts:50]: These changes shouldn't be here. Please do a rollback to the changes in this file and submit it in the same branch.

Comment (Demon-Sheriff) [reply src/app/components/navigable-story-teaser/navigable-story-teaser.component.ts:50]: Alright, sure.

**Comment by rolivencia** [comment]
> @rolivencia I have made changes in the bug_report.yml file to automatically assign the bosquejo label on new issues, please check it out and also do you want me to make changes in the readme file as well for contributor guidelines ? I will do that as well if you want that.

It won't be necessary, since the labels are self-contained on what their purpose is.

**Comment by Demon-Sheriff** [comment]
@rolivencia any updates ?

**Comment by rolivencia** [comment]
> @rolivencia any updates ?
> 

I'm waiting for your changes in the PR. Please check the comments I left yesterday.

**Comment by Demon-Sheriff** [comment]
> Hi @Demon-Sheriff, do you have any updates with this one? Just roll back the changes included in `navigable-story-teaser.component.ts` and we're good to go.

hey @rolivencia are we good to go now, sorry for taking too late :P

**Comment by rolivencia** [comment]
> > Hi @Demon-Sheriff, do you have any updates with this one? Just roll back the changes included in `navigable-story-teaser.component.ts` and we're good to go.
> 
> hey @rolivencia are we good to go now, sorry for taking too late :P

No worries, it wasn't an urgent issue. Can you apply the same change in all the issue templates? I just figured out that we need this by default in **all** the issue templates.

**Pull request merged**